src/EnergyPlus/DataHeatBalSurface.hh

```diff
@@ -267,6 +270,8 @@ struct HeatBalSurfData : BaseGlobalStruct
         this->SurfTempInsOld.deallocate();
         this->SurfTempInTmp.deallocate();
         this->SurfHcExt.deallocate();
+        this->SurfWinCoeffAdjRatioIn.deallocate();
+        this->SurfWinCoeffAdjRatioOut.deallocate();
```

Comment:
Is there a good reason why you need both these and the same variables in DataHeatBalance.hh?  Seems like should pick one place, rather than follow bad old patterns.

Reply:
Right. I'm reverting that.